### PR TITLE
card-types: code refactor

### PR DESCRIPTION
**Pull request by acodegirl**
Addressed review comments.

### CODE REVIEW

File: assets/src/console/pages/gateway-accounts/CardTypesPage.tsx [hunk 1]
```diff
@@ -37,35 +37,34 @@ interface Props {
...
-    let label =
-      cardType.brand === CardTypeBrand.Visa ||
-      cardType.brand === CardTypeBrand.MasterCard
-        ? cardType.label + " " + cardType.type.toLowerCase()
-        : cardType.label;
-    return label;
-  } else {
+  if (!cardType.type || !cardType.label) {
     return "";
+  } else {
```
Comment (jonathaningram) [inline assets/src/console/pages/gateway-accounts/CardTypesPage.tsx]: @acodegirl you don't need the else here:

```ts
if (!x) {
  return "";
}
// ...
return ...
```

File: assets/src/console/pages/gateway-accounts/CardTypesPage.tsx [hunk 2]
```diff
@@ -37,35 +37,34 @@ interface Props {
...
-  data.cardTypes
-    .filter(cardType => cardType.type === type)
-    .map(cardType => {
-      return (
-        <>
+const getCardTypesList = (data: CardTypesQuery, type: string) => {
+  if (!data) {
+    return <></>;
+  } else {
+    return data.cardTypes
```
Comment (jonathaningram) [inline assets/src/console/pages/gateway-accounts/CardTypesPage.tsx]: @acodegirl same here, no need for the else.

File: assets/src/console/pages/gateway-accounts/CardTypesPage.tsx [hunk 3]
```diff
@@ -37,35 +37,34 @@ interface Props {
...
-const getCardTypesList = (data: CardTypesQuery, type: string) =>
-  data &&
-  data.cardTypes
-    .filter(cardType => cardType.type === type)
-    .map(cardType => {
-      return (
-        <>
+const getCardTypesList = (data: CardTypesQuery, type: string) => {
+  if (!data) {
+    return <></>;
```
Comment (jonathaningram) [inline assets/src/console/pages/gateway-accounts/CardTypesPage.tsx]: @acodegirl i think you should just return null here, not a fragment

File: assets/src/console/pages/gateway-accounts/CardTypesPage.tsx (hunk 1 above)
Comment (acodegirl) [reply assets/src/console/pages/gateway-accounts/CardTypesPage.tsx]: @jonathaningram - Addressed all review comments.

**Pull request merged**